website/app/components/doc/page/header/index.hbs

```diff
@@ -10,35 +10,40 @@
   </LinkTo>
   <nav class="doc-page-header__nav-menu" aria-label="primary navigation" id="primary-navigation">
     <ul role="list" aria-labelledby="primary-navigation">
-      <Doc::Page::Header::NavItem @label="About" @route="about" @currentTopRoute={{@currentTopRoute}} />
-      <Doc::Page::Header::NavItem @label="Foundations" @route="foundations" @currentTopRoute={{@currentTopRoute}} />
       <Doc::Page::Header::NavItem @label="Components" @route="components" @currentTopRoute={{@currentTopRoute}} />
+      <Doc::Page::Header::NavItem @label="Foundations" @route="foundations" @currentTopRoute={{@currentTopRoute}} />
```

Comment:
[(personal) thought] while before the order of the items in the top navigation had some kind of semantic logic, the new order seems totally arbitrary _for me_

Reply:
also
<img width="1792" alt="modified" src="https://github.com/user-attachments/assets/728e0ca0-b7d9-498c-8657-4e0f423a6437">

Reply:
agree with @didoo here, the homepage cards match the navigation (in terms of order and number) was intentional.
I also wonder if we should consider adding a 'Content' card

Reply:
Both of these should be ordered the same way as indicated [in Figma](https://www.figma.com/design/ZlIlup8HxCOlQLXRwv56In/HDS-Content-Guidelines?node-id=119-3592&t=nytVVDuKCV7lHDmU-1).

The ordering now of the navigational items are based on which pages are visited the most (Components being first, then foundations, patterns, content and about).

Reply:
oops! I missed updating the cards, I just pushed a fix for them.